CHANGELOG.md

```diff
@@ -17,6 +17,27 @@ Possible sections in each release:
 * Security: in case of vulnerabilities.
 
 
+### [v0.13.0] - TBD
+
+IN this release we
```

Comment:
I would just remove this line

Reply:
Woops. I think I started a sentence there, but made it a bulletpoint.